### ISSUE TITLE
timus

### ISSUE THREAD

**Issue by GubHit** [tomilov/sweepline#2]
LMAO, so you're the guy that tries to pass timus 1369 since 2016 and makes cynical idiot of himself in comments???
does this voronoi pass this problem?
I'll be sure to try it out. thanks.

**Comment by tomilov** [comment]
Good try to hurt, boy. I'm unaffected. Still can't solve the problem?

**Comment by GubHit** [comment]
LMAO, LoLoLooooLoLooL. you even respond after almost 2 months - dont you have anything better to do? haha, probably - since you solve a freaking problem for 2 years :DDD, loolool, 
i still havent touched the problem, but if i will, i'll definitely use your code, thanks, loser.

**Comment by GubHit** [comment]
I'll be solving it under the name "Tomilov_Is_An_Idiot" or something like that in 2 weeks or so. Or "Tomilov_2_Year_Solve_One_problem", i dont know yet, lmaoloooloo.

**Comment by GubHit** [comment]
looolLoooL, lmao. sure. you did not pass this problem in 2016, it does not show in your submission as AC in 2016, i can only see an AC in 2018. As far as it goes: you're making a complete fool of yourself in comments and sucking lxn's d*ck for some hints about his solution while shitting on everyone else. LooLoooLool. good luck there mate, go suck some more. Lmao Looool.

What about me? dunno, maybe I'll start solving it in a few weeks or so, i'll make an account "Tomilov_2_Year_Solve_One_problem" when I'll start solving it :D, be sure to watch it, I'll be sure to use your code, lmaoo lool.

In summary:
* arrogant
* sucking off guys better than you
* condescending
* 2 years to solve one problem
In conclusion: meeeh, I'd rate you 2/5

**Comment by tomilov** [comment]
Rejudge means, that all submissions (even not accepted ones) are passed through judging system as a batch. It is how it works in general.

You have bad manners for sure, but it is just your problem, my ego is untouched, be sure.

Good luck to beat my top best awesome solution. Argh! "Arrogant", he is right. Em... just my solution. Yes.

**Comment by GubHit** [comment]
ew, why would i try to beat your top solution? it's top, but not awesome for sure... 2 years on one problem? pffft, please, I have a life, you're the one who sends one million submissions on one problem to get to the top, lmao, loooloooloool, what a kindergarten.

aand no... you didn't pass it in 2016, only in 2018 :D, it does not show in your submissions, lmaoo.

"my ego is untouched, be sure", who cares about your ego, I only care about your easy-peasy voronoi code (shit code by the way, but i hope it passes), which I thanked you for, thanks again, lmao looloool.

**Comment by tomilov** [comment]
I can conclude, that you read that topics not too thoroughly. If you'll read them again then you can understand voronoi (even mine implementation) is not on the right path to solve the problem. Constant factor for Fortune's algorithm is too large. For M = 100000 and N = 100000 is still too large to be the best choise.

**Comment by tomilov** [comment]
@GubHit By the way [author of the problem](https://twitter.com/denplusplus/status/601572081659092992) himself tried to solve the problem during 10 years in a row starting from the moment it was published on timus: more then ten attemts documented in [his github repository](https://github.com/evilmucedin/timus) (and seems more then 13 overall). In spite of this he has [slowest solution](http://acm.timus.ru/rating.aspx?space=1&num=1369&count=1000) (top 22 of 22 atm). The problem is really hard and almost hardest one among those of them which can be solved in polynomial time.

**Comment by GubHit** [comment]
@GubHit, afaik you didn't solve it with voronoi, you solved it with some floating point trickery.
I tell you what - I'll solve this problem with voronoi easy. Even more so funny - I'll solve it using YOUR voronoi, which you failed to do. How much time do you give me?

**Comment by tomilov** [comment]
@GubHit it seems that first commit was on 21 Oct 2016 and the last one was 4 Feb 2018. idk how many hours I put into the work, but it was notable amount of my spare time for sure.

**Comment by GubHit** [comment]
@tomilov, i think you didn't get me. i am telling you i can solve this problem easy-peasy and it won't take me 2 years. i just want to drop some salt on your dumb ass, you arrogant prick. what is the timeframe you give me to solve this problem?

**Comment by tomilov** [comment]
@GubHit I have no interest to put you into some constraints. Voronoi over floating point filed is always buggy. There is no way to make points strictly ordered in general. There is no transitivity in the field, when `bool operator < (float a, float b)` is `{ return a + eps < b; }` for non-zero epsilon.
Hence I don't recommend you to use any Voronoi implementation.

**Comment by GubHit** [comment]
@tomilov, this is irrelevant. i am just interested in the fact that you're a dumb arrogant prick who needs to be put in place. you're a dipshit who spent 2 years solving one problem and running around sucking off guys who solved this problem saying things like "ooh, wow, share some tips", and talking to other guys like "geez, what a noob". the best way to put you in place is to solve this problem really fast, which i can do easily. set up a timeframe for me, and I'll mop the floor with your beak of a nose, NPD boy. 
big ego trash like you should be put in place: you're not as smart as you think

**Comment by tomilov** [comment]
> talking to other guys like "geez, what a noob"
 
proof please.

I didn't think that I am smart. Do you have quotes to illustrate?

**Comment by GubHit** [comment]
here you're sucking off guys better than you:
* "I really want to see your solution! Can we trade it for top3?"
* "I interested to talk with you (I will not bother you much), can you give any contact?"

here you're calling a some guys out (who are much much better than you by the way, especially Shen Yang)
* "gz. It is called "lexicographical sorting" "
* "It bothers you? Do you want to talk about it?"
* "Bad news if it is so."

and it's quite funny, because you first called out Shen Yang, and then you sucked his dick and still failed to solve the problem with voronoi.

the challenge still stands, NPD boy.

**Comment by tomilov** [comment]
In my part of the world it is not shame to ask someone for help or (especially) for mutually beneficial trade.
I found common ground with Shen Yang, don't worry. He is cool.
What is wrong with my sentence about lexicographical sorting? Why congratulations is the part of quote? You think too badly of me. Definitely in this point I can 100% conclude you have syndrome of searching for deep meaning. Don't take it too personally, it's just a side note.
WRT "It bothers you? Do you want to talk about it?" maybe you right here.
WRT "Bad news if it is so." - again, that syndrome.
I absolutely respect that guys. You can rely on more than just your perception of my speech. You can also see how the interlocutors react to it. Our conversation is going well as you can see. Do you think that guys completely insensitive to (imaginary) insults? I don't think so.

**Comment by GubHit** [comment]
"I absolutely respect that guys" by talking like a condescending shit?
and yeah, especially "geez" part is veeery respectful, fat ego boy, i can even bet, (if you accept the challenge of course), that when i solve the problem in a matter of days you'll be sucking my dick too for some clues, won't you?

**Comment by tomilov** [comment]
Not interesting. You can do what you want. There is no such logical construction in my part world: `when i solve the problem in a matter of days you'll be sucking my dick too for some clues`. I don't think this is feasible like by wave of magic wand. This is your delusion for sure. Just children misconception. Never seen such a situation, when it works so. Why should I want to force you to do something?
I know, that this problem can be solved in a couple of hours. It does - above I said I spent 4 attempts to do so. You still continue to ignore this information. Why?

**Comment by GubHit** [comment]
suuure. more like 2 years. keep telling this to yourself. you can't even pass it with voronoi, because you can't even implement it. your code here is a shit show.
also very funny how you respond after what, like a freakin year? definitely struck some of your ego strings. anyway keep constructing these walls for yourself, because deep down you know you're an arrogant prick.
accepting a challenge even is not hard for you, all you had to do is to setup a timeframe, but you WON"T <- and this is the key point, your ego is pissed the fuck out that some anon can solve the problem in days that took you 2 years.
ping me next year when you grow some balls :)

**Comment by tomilov** [comment]
I think I have said enough to let you know that you are mistaken in almost all your statements about me.

**Comment by GubHit** [comment]
![2019-10-25_225930](https://user-images.githubusercontent.com/4038735/67600755-693aa580-f77b-11e9-9bc5-ab2f9c22d010.png)

**Comment by tomilov** [comment]
It seems the question is settled.
